packages/wonder-stuff-server-google/src/types.js

```diff
@@ -0,0 +1,155 @@
+// @flow
```

Comment:
Versions of types from `shared/types.js` in render-gateway, with some renames and the server options being updated to accommodate the split of `shared/start-gateway.js` into two pieces (the previous PR and this one).